### PR TITLE
Update CLI commands for v2.6 - v2.8

### LINKED ISSUE #198: Explain undocumented options in Rancher CLI docs
This doc issue comes from this Slack conversation:

the rancher cli has undocumented options for example `rancher up` is available in the v2.2.0 cli. But that option is not documented here https://rancher.com/docs/rancher/v2.x/en/cli/

adrian [12:04 PM]
i think it's legacy 1.6. it's analagous to `docker-compose up`
or at least it was

dan [12:08 PM]
It's for importing a cluster and projects

cjellick [12:49 PM]
@dan link that doc where it actually does explain this. It’s actually well documented just not where the rest of cli docs are I think

dan [1:11 PM]
@mak3r we refer to it as 'cloning a cluster' in the docs https://rancher.com/docs/rancher/v2.x/en/cluster-admin/cloning-clusters/ (edited) 
But I agree, the cli help needs updated

### PR DESCRIPTION
**Pull request by LucasSaintarbor**
Fixes #198 

## Description

- Update [Rancher CLI command](https://ranchermanager.docs.rancher.com/reference-guides/cli-with-rancher/rancher-cli) for v2.6 - v2.8

- Rancher CLI v2.6.11 
  Screenshot 2024-01-11 at 10 47 10 AM

- Rancher CLI v2.7.7
  Screenshot 2024-01-11 at 10 47 36 AM

- Rancher CLI v2.8.0
  Screenshot 2024-01-11 at 10 46 47 AM

### CODE REVIEW

File: docs/reference-guides/cli-with-rancher/rancher-cli.md [hunk 1]
```diff
@@ -68,18 +68,24 @@ The following commands are available for use in Rancher CLI.
...
+| `multiclusterapps, [multiclusterapp mcapps mcapp]`  | Performs operations with multi-cluster apps.  |
+| `namespaces, [namespace]`  | Performs operations on [namespaces](../../how-to-guides/new-user-guides/manage-namespaces.md).  |
+| `nodes, [node]`  | Performs operations on [nodes](../../how-to-guides/new-user-guides/manage-clusters/nodes-and-node-pools.md).  |
 | `projects, [project]`  | Performs operations on [projects](../../how-to-guides/new-user-guides/manage-clusters/projects-and-namespaces.md).  |
 | `ps`  | Displays [workloads](../../pages-for-subheaders/workloads-and-pods.md) in a project.  |
+| `server`  | Performs operations for the server.  |
 | `settings, [setting]`  | Shows the current settings for your Rancher Server.  |
 | `ssh`  | Connects to one of your cluster nodes using the SSH protocol.  |
+| `up`  | Applies compose config.  |
+| `wait`  | Waits for resoruces cluster, app, project, multiClusterApp.  |
```
Comment (martyav) [inline docs/reference-guides/cli-with-rancher/rancher-cli.md:85]:
```suggestion
| `wait`  | Waits for resources cluster, app, project, multiClusterApp.  |
```

File: docs/reference-guides/cli-with-rancher/rancher-cli.md [hunk 2]
```diff
@@ -68,18 +68,24 @@ The following commands are available for use in Rancher CLI.
...
+| `token`  | Authentictes and generates new kubeconfig token.  |
```
Comment (martyav) [inline docs/reference-guides/cli-with-rancher/rancher-cli.md:86]:
```suggestion
| `token`  | Authenticates and generates new kubeconfig token.  |
```

File: versioned_docs/version-2.6/reference-guides/cli-with-rancher/rancher-cli.md [hunk 3]
```diff
@@ -68,15 +68,22 @@ The following commands are available for use in Rancher CLI.
...
+| `multiclusterapps, [multiclusterapp mcapps mcapp]` | Performs operations with multi-cluster apps.  |
+| `namespaces, [namespace]`  | Performs operations on [namespaces](../../how-to-guides/new-user-guides/manage-namespaces.md).   |
+| `nodes, [node]`  | Performs operations on [nodes](../../how-to-guides/new-user-guides/manage-clusters/nodes-and-node-pools.md).   |
 | `projects, [project]`  | Performs operations on [projects](../../how-to-guides/new-user-guides/manage-clusters/projects-and-namespaces.md).  |
 | `ps`  | Displays [workloads](../../pages-for-subheaders/workloads-and-pods.md) in a project.  |
+| `server`  | Performs operations for the server.  |
 | `settings, [setting]`  | Shows the current settings for your Rancher Server.  |
 | `ssh`  | Connects to one of your cluster nodes using the SSH protocol.  |
+| `up`  | Applies compose config.  |
+| `wait`  | Waits for resoruces cluster, app, project, multiClusterApp.  |
```
Comment (martyav) [inline versioned_docs/version-2.6/reference-guides/cli-with-rancher/rancher-cli.md:85]:
```suggestion
| `wait`  | Waits for resources cluster, app, project, multiClusterApp.  |
```

File: versioned_docs/version-2.6/reference-guides/cli-with-rancher/rancher-cli.md [hunk 4]
```diff
@@ -68,15 +68,22 @@ The following commands are available for use in Rancher CLI.
...
+| `token`  | Authentictes and generates new kubeconfig token.  |
```
Comment (martyav) [inline versioned_docs/version-2.6/reference-guides/cli-with-rancher/rancher-cli.md:86]:
```suggestion
| `token`  | Authenticates and generates new kubeconfig token.  |
```

File: versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md [hunk 5]
```diff
@@ -68,15 +68,22 @@ The following commands are available for use in Rancher CLI.
 | `catalog`  | Performs operations on [catalogs](../../pages-for-subheaders/helm-charts-in-rancher.md).  |
 | `clusters, [cluster]`  | Performs operations on your [clusters](../../pages-for-subheaders/kubernetes-clusters-in-rancher-setup.md).  |
 | `context`  | Switches between Rancher [projects](../../how-to-guides/new-user-guides/manage-clusters/projects-and-namespaces.md). For an example, see [Project Selection](#project-selection).  |
+| `globaldns`  | Performs operations on global DNS providers and entries.  | <!-- Review -->
```
Comment (martyav) [inline versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md:71]:
```suggestion
| `globaldns`  | Performs operations on global DNS providers and entries.  | 
```

I rm'd the comment on the assumption that it was accidentally left in

File: versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md [hunk 6]
```diff
@@ -68,15 +68,22 @@ The following commands are available for use in Rancher CLI.
...
+| `multiclusterapps, [multiclusterapp mcapps mcapp]`  | Performs operations with multi-cluster apps.  |
+| `namespaces, [namespace]`  | Performs operations on [namespaces](../../how-to-guides/new-user-guides/manage-namespaces.md).  |
+| `nodes, [node]`  | Performs operations on [nodes](../../how-to-guides/new-user-guides/manage-clusters/nodes-and-node-pools.md).  |
 | `projects, [project]`  | Performs operations on [projects](../../how-to-guides/new-user-guides/manage-clusters/projects-and-namespaces.md).  |
 | `ps`  | Displays [workloads](../../pages-for-subheaders/workloads-and-pods.md) in a project.  |
+| `server`  | Performs operations for the server.  |
 | `settings, [setting]`  | Shows the current settings for your Rancher Server.  |
 | `ssh`  | Connects to one of your cluster nodes using the SSH protocol.  |
+| `up`  | Applies compose config.  |
+| `wait`  | Waits for resoruces cluster, app, project, multiClusterApp.  |
```
Comment (martyav) [inline versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md:85]:
```suggestion
| `wait`  | Waits for resources cluster, app, project, multiClusterApp.  |
```

File: versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md [hunk 7]
```diff
@@ -68,15 +68,22 @@ The following commands are available for use in Rancher CLI.
...
+| `token`  | Authentictes and generates new kubeconfig token.  |
```
Comment (martyav) [inline versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md:86]:
```suggestion
| `token`  | Authenticates and generates new kubeconfig token.  |
```

File: versioned_docs/version-2.8/reference-guides/cli-with-rancher/rancher-cli.md [hunk 8]
```diff
@@ -68,18 +68,24 @@ The following commands are available for use in Rancher CLI.
...
+| `multiclusterapps, [multiclusterapp mcapps mcapp]`  | Performs operations with multi-cluster apps.  |
+| `namespaces, [namespace]`  | Performs operations on [namespaces](../../how-to-guides/new-user-guides/manage-namespaces.md).  |
+| `nodes, [node]`  | Performs operations on [nodes](../../how-to-guides/new-user-guides/manage-clusters/nodes-and-node-pools.md).  |
 | `projects, [project]`  | Performs operations on [projects](../../how-to-guides/new-user-guides/manage-clusters/projects-and-namespaces.md).  |
 | `ps`  | Displays [workloads](../../pages-for-subheaders/workloads-and-pods.md) in a project.  |
+| `server`  | Performs operations for the server.  |
 | `settings, [setting]`  | Shows the current settings for your Rancher Server.  |
 | `ssh`  | Connects to one of your cluster nodes using the SSH protocol.  |
+| `up`  | Applies compose config.  |
+| `wait`  | Waits for resoruces cluster, app, project, multiClusterApp.  |
```
Comment (martyav) [inline versioned_docs/version-2.8/reference-guides/cli-with-rancher/rancher-cli.md:85]:
```suggestion
| `wait`  | Waits for resources cluster, app, project, multiClusterApp.  |
```

File: versioned_docs/version-2.8/reference-guides/cli-with-rancher/rancher-cli.md [hunk 9]
```diff
@@ -68,18 +68,24 @@ The following commands are available for use in Rancher CLI.
...
+| `token`  | Authentictes and generates new kubeconfig token.  |
```
Comment (martyav) [inline versioned_docs/version-2.8/reference-guides/cli-with-rancher/rancher-cli.md:86]:
```suggestion
| `token`  | Authenticates and generates new kubeconfig token.  |
```

**Review by martyav — CHANGES_REQUESTED** [review]
Some typo fixes

File: versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md [hunk 10]
```diff
@@ -68,15 +68,22 @@ The following commands are available for use in Rancher CLI.
...
+| `globaldns`  | Performs operations on global DNS providers and entries.  | <!-- Review -->
 | `inspect [OPTIONS] [RESOURCEID RESOURCENAME]`  | Displays details about [Kubernetes resources](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#resource-types) or Rancher resources (i.e.: [projects](../../how-to-guides/new-user-guides/manage-clusters/projects-and-namespaces.md) and [workloads](../../pages-for-subheaders/workloads-and-pods.md)). Specify resources by name or ID.  |
-| `kubectl`  |Runs [kubectl commands](https://kubernetes.io/docs/reference/kubectl/overview/#operations).   |
+| `kubectl`  | Runs [kubectl commands](https://kubernetes.io/docs/reference/kubectl/overview/#operations).   |
 | `login, [l]`  | Logs into a Rancher Server. For an example, see [CLI Authentication](#cli-authentication).  |
-| `namespaces, [namespace]`  |Performs operations on namespaces.   |
-| `nodes, [node]`  |Performs operations on nodes.   |
+| machines, [machine]  | Performs operations on machines.  |
```
Comment (btat) [inline versioned_docs/version-2.7/reference-guides/cli-with-rancher/rancher-cli.md:75]:
```suggestion
| `machines, [machine]`  | Performs operations on machines.  |
```

**Review by btat — CHANGES_REQUESTED** [review]

**Comment by LucasSaintarbor** [comment]
@btat @martyav Thanks for reviewing! Added the changes.

**Pull request merged**